docs/source/status.rst

```diff
@@ -156,13 +156,13 @@ Flight RPC
 +-----------------------------+-------+-------+-------+------------+-------+-------+-------+
 | gRPC + TLS transport        | ✓     | ✓     | ✓     |            | ✓     |       |       |
 +-----------------------------+-------+-------+-------+------------+-------+-------+-------+
-| RPC error codes             | ✓     | ✓     |       |            | ✓     |       |       |
+| RPC error codes             | ✓     | ✓     | ✓     |            | ✓     |       |       |
 +-----------------------------+-------+-------+-------+------------+-------+-------+-------+
 | Authentication handlers     | ✓     | ✓     | ✓     |            | ✓ (2) |       |       |
 +-----------------------------+-------+-------+-------+------------+-------+-------+-------+
-| Custom client middleware    | ✓     | ✓     |       |            |       |       |       |
+| Custom client middleware    | ✓     | ✓     | ✓     |            |       |       |       |
```

Comment:
i think this was brought up no the mailing list.  For new feature we might want to start recording verson numbers that a feature was availalable int (or maybe add this as a foot-note.

Reply:
That's gonna be a bit difficult until the CI is updated to create the necessary tags for the golang libraries, I believe there's a JIRA card for it, I just don't know enough about that side of the CI scripts to know what to change and where in order to add the correct formatted tags for the go versions